src/services/SeedService.ts

```diff
@@ -27,6 +27,7 @@ export interface ISeedCreatedEventArgs {
 export class SeedService {
 
   public seeds: Map<Address, Seed>;
+  public static seedFee = 0;
```

Comment:
lets make this 0.0.  Add a code comment "If the value is ever > 0, then should be a fraction like 0.1 to represent 1%".